tensorflow/compiler/xla/service/gpu/BUILD

```diff
@@ -907,30 +909,32 @@ cc_library(
 
 cc_library(
     name = "gemm_algorithm_picker",
-    srcs = if_cuda_is_configured(["gemm_algorithm_picker.cc"]),
-    hdrs = if_cuda_is_configured(["gemm_algorithm_picker.h"]),
-    deps = if_cuda_is_configured([
+    srcs = ["gemm_algorithm_picker.cc"],
+    hdrs = ["gemm_algorithm_picker.h"],
+    deps = [
         ":backend_configs_cc",
         ":buffer_comparator",
         ":gemm_thunk",
         ":gpu_asm_opts_util",
         ":gpu_conv_runner",
         ":ir_emission_utils",
         ":stream_executor_util",
-        "@com_google_absl//absl/types:optional",
+        "//tensorflow/compiler/tf2xla:common",
```

Comment:
Why do we need this dependency?

Reply:
This is needed for `EncodePrimitiveTypeAsDataType`, called in `DoBlasPlansAutoTune`, which converts `tensorflow::PrimitiveType` to `tensorflow::DataType`.